dkist/tests/test_benchmarks.py

```diff
@@ -35,3 +32,19 @@ def plot_and_save_fig(ds=visp_dataset_no_headers, axes=axes):
         ds.plot(plot_axes=axes)
         plt.savefig("tmpplot")
         plt.close()
+
+
+@pytest.mark.benchmark
+def test_dataset_compute_data_full_files(benchmark, real_visp):
+    ds = real_visp[0, :15]
+    benchmark(ds.data.compute)
```

Comment:
Given what we learnt about IO and benchmarking in https://github.com/DKISTDC/dkist/pull/399#issuecomment-2220534572 I think we should add a comment or docstring here noting that this is timing this without the IO overhead.

I think that it's still a useful benchmark, i.e. it would have caught #361 but we should make sure we don't forget what it is actually testing while we are using codspeed.